src/applications/pensions/config/chapters/04-household-information/dependentChildrenPages.js

```diff
@@ -0,0 +1,423 @@
+    'view:disabilityDocs': {
+      'ui:description': DisabilityDocsAlert,
+      'ui:options': {
+        expandUnder: 'disabled',
+      },
+    },
+    'view:disabilityInformation': {
+      'ui:description': DependentSeriouslyDisabledDescription,
+    },
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      disabled: yesNoSchema,
+      'view:disabilityDocs': { type: 'object', properties: {} },
+      'view:disabilityInformation': { type: 'object', properties: {} },
+    },
+    required: ['disabled'],
+  },
+};
+
+/** @returns {PageSchema} */
+const previouslyMarriedPage = {
+  uiSchema: {
+    ...arrayBuilderItemSubsequentPageTitleUI(
+      ({ formData }) =>
+        formData?.fullName
+          ? `${formatFullName(formData.fullName)} marriage information`
+          : 'Marriage information',
+    ),
+    previouslyMarried: yesNoUI({
+      title: 'Has your child ever been married?',
+    }),
+    married: yesNoUI({
+      title: 'Are they currently married?',
+      expandUnder: 'previouslyMarried',
+      required: (formData, index) =>
+        get(['dependents', index, 'previouslyMarried'], formData),
+    }),
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      previouslyMarried: yesNoSchema,
+      married: yesNoSchema,
+    },
+    required: ['previouslyMarried'],
+  },
+};
+
+/** @returns {PageSchema} */
+const inHouseholdPage = {
+  uiSchema: {
+    ...arrayBuilderItemSubsequentPageTitleUI(
+      ({ formData }) =>
+        formData?.fullName
+          ? `${formatFullName(formData.fullName)} household information`
+          : 'Household information',
+    ),
+    childInHousehold: yesNoUI({
+      title: 'Does your child live with you?',
+    }),
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      childInHousehold: yesNoSchema,
+    },
+    required: ['childInHousehold'],
+  },
+};
+
+/** @returns {PageSchema} */
+const addressPage = {
+  uiSchema: {
+    ...arrayBuilderItemSubsequentPageTitleUI(
+      ({ formData }) =>
+        formData?.fullName
+          ? `${formatFullName(formData.fullName)} address information`
+          : 'Address information',
+    ),
+    childAddress: addressUI({
+      omit: ['isMilitary', 'street3'],
+    }),
+    personWhoLivesWithChild: merge(
+      {},
+      {
+        'ui:title': 'Who do they live with?',
+      },
+      fullNameUI(),
+    ),
+    monthlyPayment: merge(
+      {},
+      currencyUI(
+        "How much do you contribute per month to your child's support?",
+      ),
+      {
+        'ui:options': {
+          classNames: 'schemaform-currency-input-v3',
+        },
+      },
+    ),
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      childAddress: addressSchema({ omit: ['isMilitary', 'street3'] }),
+      personWhoLivesWithChild: fullNameSchema,
+      monthlyPayment: { type: 'number' },
+    },
+    required: ['childAddress', 'monthlyPayment'],
+  },
+};
+
+export const dependentChildrenPages = arrayBuilderPages(
```

Comment:
quite a few pages! some enhancements we for this pattern we might consider when using it for more pages - should we show all the information  on the final review and submit page? e.g. all the item pages vs just the cards. And also currently we have to go through every page again if we edit from the review page. we could also consider enhancing that to just correct the particular page.